src/features/_featureTemplate/actions.js

```diff
@@ -28,20 +28,12 @@ const
     return { [CALL_API]: obj }
   });
 
-export const
```

Comment:
This export const should remain so we can avoid typing it over and over. Unless you think the nil = null; at the end to terminate it will cause issues.